src/index.ts

```diff
@@ -12,6 +13,13 @@ export interface ReadStreamOptions {
   encoding?: ReadableOptions["encoding"];
 }
 
+// Work around node's MaxListenersExceededWarning for highly concurrent
+// workloads by creating a "proxy" event emitter. See:
+// https://github.com/mike-marcacci/fs-capacitor/issues/30
+const processExitProxy = new EventEmitter();
+processExitProxy.setMaxListeners(Infinity);
+process.addListener("exit", () => processExitProxy.emit("exit"));
```

Comment:
```suggestion
process.once("exit", () => processExitProxy.emit("exit"));
```

https://nodejs.org/dist/latest-v15.x/docs/api/events.html#events_emitter_once_eventname_listener

Reply:
Ah yes, haha. I suppose in this case `once` and `on`/`addListener` are actually comparable (since `exit` is by definition a singleton event).